kuka_iiqka_eac_driver/src/hardware_interface.cpp

```diff
@@ -270,6 +411,19 @@ return_type KukaEACHardwareInterface::write(const rclcpp::Time &, const rclcpp::
   robot_ptr_->GetControlSignal().AddTorqueValues(hw_torque_commands_);
   robot_ptr_->GetControlSignal().AddStiffnessAndDampingValues(
     hw_stiffness_commands_, hw_damping_commands_);
+    
+    hw_twist_commands_vector_[0]=hw_twist_commands_.linear.x;
```

Comment:
Are we sure these copies are needed? I think the controller should do this

Reply:
Agreed, you currently store the same data twice in different structures, but the command interfaces exported will be a simple vector in all cases (with names making everything straightforward)
so you can just add the vector at the command interface configuration, the `hw_twist_commands_` is not needed